src/server_manager/install_scripts/install_server.sh

```diff
@@ -260,6 +260,7 @@ function start_shadowbox() {
   # rather than pass in the environment.
   declare -a docker_shadowbox_flags=(
     --name shadowbox --restart=always --net=host
+    --label=com.centurylinklabs.watchtower.enable=true
```

Comment:
A disadvantage of this approach is that existing shadowbox containers won't get the label unless there's a manual re-install.  

Is it possible we could add the label directly to the Dockerfile?  Then when we next  push shadowbox all the existing containers will get this label automatically.

Reply:
Indeed...  if stable image is created from `src/shadowbox/docker/Dockerfile`, perhaps something similar might help:
```
diff --git src/shadowbox/docker/Dockerfile src/shadowbox/docker/Dockerfile
index f5c1ec6..9601f42 100644
--- src/shadowbox/docker/Dockerfile
+++ src/shadowbox/docker/Dockerfile
@@ -24,2 +24,5 @@ LABEL shadowbox.outline-ss-server_version="${SS_VERSION}"

+# Set watchtower to automatically fetch docker image updates
+LABEL com.centurylinklabs.watchtower.enable=true
+
 ARG GITHUB_RELEASE
```
And label here would also be used for custom `SB_IMAGE` but not sure if that is desired behaviour 🤔 maybe it'll be local in most cases...

Reply:
Moved the label to Dockerfile